db/migrate/20150110084027_change_team_slug_to_citext.rb

```diff
@@ -0,0 +1,5 @@
+class ChangeTeamSlugToCitext < ActiveRecord::Migration
+  def change
+    change_column :teams, :slug, :citext, null: false, index: true , unique: true
```

Comment:
the schema.rb won't change. I did not add another migration. 
Schema.rb contain the timestamp of last migration .